python/ccxt/test/test_sync.py

```diff
@@ -363,6 +367,19 @@ def test_balance(exchange):
 
 
 def test_symbol(exchange, symbol, code):
+    if not argv.privateOnly:
+        run_public_tests(exchange, symbols, codes)
+
+    if argv.privateOnly or argv.private:
```

Comment:
@ttodua fyi: I did this small refactor so we can have more control over how to run the private tests, and now by default we only run the public ones unless we explicitly provide the `private` or `privateOnly` flag. I'm also working on something that will allow us to easily run private tests safely. I'll let you know soon

Reply:
@carlosmiei iirc, this might not work or break things, because as I remember, it was not possible to work with args inside test/js/py/php files because they were manipulated (in run-tests.js) before reaching to those files. I had tried that before and I have met a dead-end and gave it up (it needed some rework in run-tests and other related files). I'll find the specific details and let you know

Reply:
@ttodua can you please elaborate a bit more? what do you think will break with these changes?
I tested and seems to be working fine

Reply:
i.e. here see this: https://github.com/ccxt/ccxt/blob/master/run-tests.js#L38 
so, where will `private[Only]` flag will go through? it's intercepted as "exchange" per that logic.

then here https://github.com/ccxt/ccxt/blob/master/run-tests.js#L150 you see what are being passed as "args" into language-specific test files.

So, i dont think this will work that way (i doubt Igor might prematurely merged the PR), but maybe you have more context and there is no issue at all and I might be wrong.

Reply:
@ttodua I think you're right, I was calling the tests files directly, so I didn't realize that, but first, this is not a breaking change because we're not using those flags by default, and second I think we can quickly fix it, I will take a look now.

Reply:
@ttodua I fixed that here: https://github.com/ccxt/ccxt/pull/16714